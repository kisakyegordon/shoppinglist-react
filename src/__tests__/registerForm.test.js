import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import RegisterForm from '../components/register/registerForm';

let match = {
    params : '1'
}

const setUp = () => {
    const props = {
        match
    };
    return shallow(<RegisterForm {...props} />);
};


describe('<RegisterForm />', () => {
    it('match a snapshot', () => {
        const wrapper = setUp();
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

});