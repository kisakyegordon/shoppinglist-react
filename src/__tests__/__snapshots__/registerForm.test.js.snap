// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterForm /> match a snapshot 1`] = `
<div>
  <ToastContainer
    autoClose={5000}
    bodyClassName=""
    className={null}
    closeButton={<DefaultCloseButton />}
    closeOnClick={true}
    hideProgressBar={true}
    newestOnTop={false}
    pauseOnHover={true}
    position="top-right"
    progressClassName=""
    style={null}
    toastClassName=""
    transition={[Function]}
  />
  <form
    onSubmit={[Function]}
  >
    <div
      style={
        Object {
          "display": "inline-block",
          "marginBottom": "30px",
        }
      }
    >
      <TextField
        disabled={false}
        floatingLabelFixed={false}
        floatingLabelText="Email"
        fullWidth={false}
        multiLine={false}
        name="email"
        onChange={[Function]}
        rows={1}
        type="text"
        underlineShow={true}
        value=""
      />
      <TextField
        disabled={false}
        floatingLabelFixed={false}
        floatingLabelText="Country/Town"
        fullWidth={false}
        multiLine={false}
        name="country_town"
        onChange={[Function]}
        rows={1}
        type="text"
        underlineShow={true}
        value=""
      />
      <TextField
        disabled={false}
        floatingLabelFixed={false}
        floatingLabelText="Password"
        fullWidth={false}
        multiLine={false}
        name="password"
        onChange={[Function]}
        rows={1}
        type="password"
        underlineShow={true}
        value=""
      />
    </div>
    <div>
      <RaisedButton
        disabled={false}
        fullWidth={false}
        label="Submit"
        labelPosition="after"
        primary={true}
        secondary={false}
        type="submit"
      />
    </div>
  </form>
</div>
`;
